misk/src/main/kotlin/misk/web/ServletHttpCall.kt

```diff
@@ -77,6 +78,10 @@ internal data class ServletHttpCall(
     this.webSocket = webSocket
   }
 
+  override fun initWebSocketListener(webSocketListener: WebSocketListener) {
+    this.upstreamResponse.initWebSocketListener(webSocketListener)
```

Comment:
The `FakeHttpCall` has a validation that init is being called only once. 
```check(this.webSocketListener == null) { "web socket listener already set" }```

Should we put one here?